frontend/lib/identification/util/card_info_utils.dart

```diff
@@ -19,3 +19,14 @@ extension Hashing on CardInfo {
     return utf8.encode(canonicalizedJsonString);
   }
 }
+
+bool isCardExpired(CardInfo cardInfo) {
+  final expirationDay = cardInfo.hasExpirationDay() ? cardInfo.expirationDay : null;
+  return expirationDay == null
+      ? false
+      : DateTime.fromMillisecondsSinceEpoch(0).add(Duration(days: expirationDay)).isBefore(DateTime.now());
+}
+
+bool isVerificationCheckExpired(String lastCheck) {
+  return DateTime.now().toUtc().isAfter(DateTime.parse(lastCheck).add(Duration(days: 7)));
```

Comment:
We should probably also check that `now` is greater or equal lastVerificationTimeStamp

Reply:
i updated this to check and added a tolerance of 
```js
// 7days x 24h + 1day for expiration day and 12h for UTC+12 = 192
int cardValidationExpireHours = 192;
```